modules/javafx.media/src/main/native/gstreamer/plugins/mfwrapper/mfwrapper.cpp

```diff
@@ -725,9 +692,7 @@ static GstFlowReturn mfwrapper_deliver_sample(GstMFWrapper *decoder, IMFSample *
     if (SUCCEEDED(hr) && cbCurrentLength > 0)
     {
         GstBuffer *pGstBuffer = gst_buffer_new_allocate(NULL, cbCurrentLength, NULL);
-        GstMapInfo info;
-
-        if (!gst_buffer_map(pGstBuffer, &info, GST_MAP_WRITE))
+        if (pGstBuffer == NULL || !gst_buffer_map(pGstBuffer, &info, GST_MAP_WRITE))
         {
             pMediaBuffer->Unlock();
             gst_buffer_unref(pGstBuffer); // INLINE - gst_buffer_unref()
```

Comment:
Is it safe to call gst_buffer_unref with `NULL`?

Reply:
No, it is not safe. I fixed it.